src/Tribe/Attendees.php

```diff
@@ -820,6 +825,47 @@ public function maybe_generate_csv() {
 		}
 	}
 
+	/**
+	 * Admin requested a CSV export from the attendees table.
+	 * Generates the download and finishes the execution.
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function export_csv_for_all() {
+		if ( empty( $_GET['attendees_csv'] ) || empty( $_GET['attendees_csv_nonce'] ) || ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+
+		// Generate filtered list of attendees.
+		$items = $this->generate_filtered_list( 'all' );
+
+		// Sanitize items for CSV usage.
+		$items = $this->sanitize_csv_rows( $items );
+
+		if ( ! empty( $items ) ) {
+			$charset  = get_option( 'blog_charset' );
+			$filename = sanitize_file_name( __( 'All event attendees', 'event-tickets' ) );
+
+			// Output headers so that the file is downloaded rather than displayed.
+			header( "Content-Type: text/csv; charset=$charset" );
```

Comment:
While not strictly necessary, I find that including brackets around interpolated variables helps with readability.

```suggestion
			header( "Content-Type: text/csv; charset={$charset}" );
```